### PR TITLE
Restructure solution for better validation of projections

### PR DESCRIPTION
**Pull request by Scottj1s**
This is also set up for automatic projection generation, based on referenced winmds.

### CODE REVIEW

File: cswinrt/cswinrt.nuspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>Windows</id>
+    <version>$version$</version>
+    <description>Windows</description>
+    <authors>cswinrt</authors>
+    <dependencies>
+      <!--<group targetFramework=".NETStandard2.0" />-->
```
Comment (manodasanW) [inline cswinrt/cswinrt.nuspec:9]: Is this removing netstandard2.0 from nuget?

File: Projections/Win2D/Win2D.csproj [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      <ReferencePathWithRefAssemblies Remove="@(ReferencePathWithRefAssemblies)" Condition="%(ReferencePathWithRefAssemblies.Filename) == 'Microsoft.Graphics.Canvas'" />
+    </ItemGroup>
+  </Target>
+
+  <!--<Target Name="CreatePackage" AfterTargets="Build" Condition="'$(IsInnerBuild)' != 'true'" Inputs="$(OutDir)\netstandard2.0\Win2D.dll;$(OutDir)\netcoreapp5.0\Win2D.dll" Outputs="$(NugetBuildDir)Win2D.$(NugetTestVersion).nupkg">
+    <PropertyGroup>
+      <NugetCommandLine>$(SolutionDir).nuget\nuget pack Win2D.nuspec -Version $(NugetTestVersion) -OutputDirectory $(NugetBuildDir) -Properties Configuration=$(Configuration);Platform=$(Platform);Win2DVersion=$(Win2DVersion) -NonInteractive -Verbosity Detailed -NoPackageAnalysis</NugetCommandLine>
+    </PropertyGroup>
+    <Exec Command="$(NugetCommandLine)" />
+    <Exec Command="rd /s/q &quot;$(NugetCacheDir)Win2D\$(NugetTestVersion)&quot;" />
```
Comment (manodasanW) [inline Projections/Win2D/Win2D.csproj:81]: I am assuming this is commented out intentionally?

**Comment by manodasanW** [comment]
Do you need to update cswinrt/packages.config for updated Microsoft.Windows.WinMD version?

File: Projections/Win2D/Win2D.csproj (hunk 2 above)
Comment (Scottj1s) [reply Projections/Win2D/Win2D.csproj:81]: this folder is cruft - deleted

**Comment by Scottj1s** [comment]
> Do you need to update cswinrt/packages.config for updated Microsoft.Windows.WinMD version?

not that I'm aware of (not waiting on any fixes)

**Pull request merged**